### PR TITLE
[#13338] Enable server logs when running quarkus integration-tests

### LINKED ISSUE #13338: Enable server logs when running quarkus integration-tests
### Description

Currently no logs are output which makes it difficult to debug issues locally and on the CI, we should enable logs by default.

### Implementation ideas

_No response_

### PR DESCRIPTION
**Pull request by ryanemerson**
Closes #13338

### CODE REVIEW

File: quarkus/integration-tests/server/src/test/resources/configuration/log4j2.xml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+            <PatternLayout pattern="${accessLogPattern}"/>
+        </RollingFile>
+    </Appenders>
+
+    <Loggers>
+        <Root level="INFO">
+            <AppenderRef ref="STDOUT" level="TRACE"/>
+
+            <!-- Uncomment just one of the two lines bellow to use alternatively JSON logging or plain-text logging to file-->
+            <AppenderRef ref="FILE"/>
```
Comment (wburns) [inline quarkus/integration-tests/server/src/test/resources/configuration/log4j2.xml:64]: Should we not swap these around so TRACE goes to the file instead and the console is INFO?

Comment (ryanemerson) [reply quarkus/integration-tests/server/src/test/resources/configuration/log4j2.xml:64]: Maybe... I just copied this file from `server/tests`. So any changes should probably be applied there as well.

Comment (wburns) [reply quarkus/integration-tests/server/src/test/resources/configuration/log4j2.xml:64]: Hrmm.. I guess it is okay then. I was more thinking looking at the logs would be better and not spam the console, but this should be okay.

**Review by wburns — COMMENTED** [review]
Only thing is I would suggest putting a mention in https://github.com/infinispan/infinispan/blob/main/quarkus/README.md so people can be aware how to tune the TRACE if needed. I know I get lost with the server/tests and have to look it up every time.

**Comment by wburns** [comment]
Integrated into main, thanks @ryanemerson !

**Pull request merged**